authz-module/index.md

```diff
@@ -0,0 +1,237 @@
+:::
+
+Run the following commands to configure the `simd` binary.
+
+Set the chain ID:
+
+```bash
+simd config chain-id demo
+```
+
+Set the keyring backend:
+
+```bash
+simd config keyring-backend test
+```
+
+## Key Setup
+
+Create a couple of test keys. Use the `--recover` option so that the addresses used in the following example commands are consistent.
+
+Add a key for Alice:
+
+```bash
+simd keys add alice --recover
+```
+
+Enter the following mnemonic:
+
+```bash
+plunge hundred health electric victory foil marine elite shiver tonight away verify vacuum giant pencil ocean nest pledge okay endless try spirit special start
+```
+
+Add a key for Bob:
+
+```bash
+simd keys add bob --recover
+```
+
+Enter the following mnemonic:
+
+```bash
+shuffle oppose diagram wire rubber apart blame entire thought firm carry swim old head police panther lyrics road must silly sting dirt hard organ
+```
+
+## Chain Setup
+
+The following commands set up a chain using the simulation application (`simapp`).
+
+Initialize the node:
+
+```bash
+simd init test --chain-id demo
+```
+
+Add Alice and an initial balance to the genesis file:
+
+```bash
+simd add-genesis-account alice 5000000000stake --keyring-backend test
+```
+
+Add Bob and an initial balance to the genesis file:
+
+```bash
+simd add-genesis-account bob 5000000000stake --keyring-backend test
+```
+
+Generate a transaction to add Alice to the initial validator set:
+
+```bash
+simd gentx alice 1000000stake --chain-id demo
+```
+
+Add the validator transaction to the genesis file:
+
+```bash
+simd collect-gentxs
+```
+
+## Start Chain
+
+You are now ready to start a single node network on your local machine.
+
+Start the chain:
+
+```bash
+simd start
+```
+
+## Submit Proposal
+
+To demonstrate an authorization to vote on a governance proposal, you must first create a governance proposal. The following command creates a text proposal that includes the minimun deposit, which allows the governance proposal to immediately enter the voting period. For more information about the command and the flag options, run `simd tx gov submit-proposal --help`.
+
+Create proposal:
+
+```bash
+simd tx gov submit-proposal --title="Test Authorization" --description="Is Bob authorized to vote?" --type="Text" --deposit="10000000stake" --from alice
+```
+
+View proposal:
+
+```bash
+simd q gov proposal 1
+```
+
+## Grant Authorization
+
+Next, the granter must `grant` an authorization to the grantee.
+
+The authorization is a "generic" authorization, which means there are no limits on the authorization. Other [authorization types](https://docs.cosmos.network/v0.44/modules/authz/01_concepts.html#built-in-authorizations) include "send", "delegate", "unbond", and "redelegate" in which case a limit on the amount of tokens can be set by the granter. In this case, the grantee is able to vote as many times as they want until the granter revokes the authorization.
```

Comment:
Small suggestion for clarity.

Also, just wondering, but "delegate", "unbond" and "redelegate" aren't listed in the built-in authorizations overview. Does [that doc](https://github.com/cosmos/cosmos-sdk/blob/master/x/authz/spec/01_concepts.md#Built-in-Authorizations) need to be updated?


```suggestion
The authorization is a "generic" authorization, which is one that takes a message type (such as `MsgVote`) as a parameter and allows the grantee unlimited authorization to execute that message on behalf of the granter. Other [authorization types](https://docs.cosmos.network/v0.44/modules/authz/01_concepts.html#built-in-authorizations) include "send", "delegate", "unbond", and "redelegate" in which case a limit on the amount of tokens can be set by the granter. In this case, the grantee is able to vote as many times as they want until the granter revokes the authorization.
```

Reply:
> Also, just wondering, but "delegate", "unbond" and "redelegate" aren't listed in the built-in authorizations overview. Does that doc need to be updated?

Yup, looks like `StakeAuthorization` needs to be added. `StakeAuthorization` defines authorization for delegate, unbond, and redelegate, which are listed out here as the options presented when using the CLI.